azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -210,28 +213,64 @@ private void createLogger() {
               + this.jobId;
       logger = Logger.getLogger(loggerName);
 
-      // Create file appender
-      String logName = createLogFileName(node);
-      logFile = new File(workingDir, logName);
-
-      String absolutePath = logFile.getAbsolutePath();
+      // Attempt to create appenders as necessary
+      attachFileAppender();
 
-      jobAppender = null;
-      try {
-        RollingFileAppender fileAppender =
-            new RollingFileAppender(loggerLayout, absolutePath, true);
-        fileAppender.setMaxBackupIndex(jobLogBackupIndex);
-        fileAppender.setMaxFileSize(jobLogChunkSize);
-        jobAppender = fileAppender;
-        logger.addAppender(jobAppender);
-        logger.setAdditivity(false);
-      } catch (IOException e) {
-        flowLogger.error("Could not open log file in " + workingDir
-            + " for job " + this.jobId, e);
+      if (props.getBoolean("azkaban.job.kafkalogging.enabled", false)) {
```

Comment:
Many of the literal strings in this file are not stored as constants somewhere. Would it look right to add this in?